examples/horovod/tensorflow-mnist-elastic.yaml

```diff
@@ -18,9 +18,9 @@ spec:
             args:
             - -np
             - "2"
-            - min-np
+            - --min-np
             - "1"
-            - max-np
+            - --max-np
```

Comment:
I am wondering how to support autoscale with HPA in the future. If we change the spec.replicas to 100, what will happen?